packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -105,33 +105,47 @@ export async function* AnthropicChatModel(
   });
   yield AI.AppendOnlyStream;
   const messages = await Promise.all(
-    messageElements.filter(AI.isElement).map(async (message) => {
-      switch (message.tag) {
-        case UserMessage:
-          return `${AnthropicSDK.HUMAN_PROMPT}: ${await render(message)}`;
-        case AssistantMessage:
-          return `${AnthropicSDK.AI_PROMPT}: ${await render(message)}`;
-        case SystemMessage:
-          throw new AIJSXError(
-            'Anthropic models do not support SystemMessage. Change your user message to instruct the model what to do.',
-            ErrorCode.AnthropicDoesNotSupportSystemMessage,
-            'user'
-          );
-        case FunctionCall:
-        case FunctionResponse:
-          throw new AIJSXError(
-            'Anthropic models do not support functions.',
-            ErrorCode.AnthropicDoesNotSupportFunctions,
-            'user'
-          );
-        default:
-          throw new AIJSXError(
-            `ChatCompletion's prompts must be UserMessage or AssistantMessage, but this child was ${message.tag.name}`,
-            ErrorCode.ChatCompletionUnexpectedChild,
-            'internal'
-          );
-      }
-    })
+    messageElements
+      .filter(AI.isElement)
+      .flatMap((message) => {
+        if (message.tag === SystemMessage) {
+          return [
+            <UserMessage>For subsequent replies you will adhere to the following instructions: {message}</UserMessage>,
+            <AssistantMessage>Okay, I will do that.</AssistantMessage>,
+          ];
+        }
+
+        return message;
+      })
+      .map(async (message) => {
+        switch (message.tag) {
+          case UserMessage:
+            return `${AnthropicSDK.HUMAN_PROMPT}:${message.props.name ? ` (${message.props.name})` : ''} ${await render(
```

Comment:
Can you add an example of using a `name` prop?

Reply:
The debate example uses it, which is what I was testing against